packages/generator/src/generator-cli.ts

```diff
@@ -29,7 +31,21 @@ export function parseCmdArgs(): GeneratorOptions {
     .config(
       'config',
       'Instead of specifying the options on the command line, you can also provide a path to single .json file holding these options. ' +
-        'The file must be a valid .json file where the keys correspond to the command line flags without dashes. Paths will be interpreted relative to the config file.'
+        'The file must be a valid .json file where the keys correspond to the command line flags without dashes. Paths will be interpreted relative to the config file.',
+      configPath => {
```

Comment:
I think there was a method which is executed while parsing (called resolve or so). Perhaps you can add this logic to this `resolve` method.

Reply:
But this method here will only be called when there is a config file. The coercion functions on the options are called in both cases.